modules/op-configuring-results.adoc

```diff
@@ -0,0 +1,8 @@
+[id="configuring-results"]
```

Comment:
Does this need a mod docs content type and a `_{context}` after the ID? And what assembly it's included in?

per:

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata